pages/docs/adapters/calculix/adapter-calculix-get-calculix.md

```diff
@@ -129,7 +129,8 @@ The source code is now in the `~/CalculiX/ccx_VERSION/src` directory. The adapte
 
 ### Building the "vanilla" CalculiX (optional)
 
-If you want to build the "vanilla" (i.e. without preCICE) CalculiX, you can now run `make` inside the `src/` directory. Depending on how you installed the dependencies above (using `apt` or from source), you might get compilation errors, such as. Often these errors can be easily fixed by modifying CalculiX `Makefile`. Please refer to [our adapter's makefile options](https://www.precice.org/adapter-calculix-get-adapter.html#makefile-options) for a list of library and include flag you might have to set depending on your installation procedure.
+If you want to build the "vanilla" (i.e. without preCICE) CalculiX, you can now run `make` inside the `src/` directory. Depending on how you installed the dependencies above (using `apt` or from source), you might get compilation errors, such as `spooles.h:26:10: fatal error: misc.h: No such file or directory
+`. Often these errors can be easily fixed by modifying CalculiX `Makefile`. Please refer to [our adapter's makefile options](https://www.precice.org/adapter-calculix-get-adapter.html#makefile-options) for a list of library and include flag you might have to set depending on your installation procedure.
```

Comment:
I would not break the line in an inline code environment:

```suggestion
If you want to build the "vanilla" (i.e. without preCICE) CalculiX, you can now run `make` inside the `src/` directory. Depending on how you installed the dependencies above (using `apt` or from source), you might get compilation errors, such as `spooles.h:26:10: fatal error: misc.h: No such file or directory`. Often these errors can be easily fixed by modifying CalculiX `Makefile`. Please refer to [our adapter's makefile options](https://www.precice.org/adapter-calculix-get-adapter.html#makefile-options) for a list of library and include flag you might have to set depending on your installation procedure.
```

If you prefer to keep the line break, then better move it after the dot.

Reply:
Also, when you are adding URLs in a page inside the the `precice.github.io` repository, it is better to skip the domain name:

`https://www.precice.org/adapter-calculix-get-adapter.html#makefile-options` --> `adapter-calculix-get-adapter.html#makefile-options`

Reasons are that the domain can change, even in smaller parts: we now, e.g. prefer the form without the `www.`. In the past, we changed everything from `http` to `https`. Redirections may apply. It is better to just leave this to the server. :wink:

When you are editing a file in the tutorials repository, then use absolute URLs (as the tutorials should also be readable in a different context, outside of the website).

Reply:
After first accidentally messing up the commit, I fixed this now in 6c71e6d.

@MakisH  If I understand you correctly: If `https://www.precice.org` appears in any link of this repo, it's wrong. Is this rule something that we can put into a test? I find the new tests for dead links etc. extremely useful. If we could also use something similar to avoid unnecessarily adding absolute links starting with `https://www.precice.org...` I imagine this might be helpful, as well.

Reply:
I don't think it would be easy to check or enforce the format of the links. I want to add the same link-checking workflow we already have in the tutorials, but this needs time, as I will need to fix any broken links and add exceptions. It's on my list.